.github/ISSUE_TEMPLATE/bug-repport.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug Report
+about: template to report a bug
+title: "Bug: [BUG NAME]"
+labels: ''
+assignees: ''
+---
+
+### Basic information
+
+* On-premises: Yes/No
+* Cloud-Provider (e.g. AWS, GCP, etc): TODO
+* Installation type: demo/prod
+* Kubernetes version: TODO
+* related Technology + Version (Ansible, Helm, Terraform): TODO
+
+
+### What is the expected result?
+
+TODO
+
+
+### What is the actual result?
+
+TODO
+
+
+### How to reproduce the issue?
+
+TODO
```

Comment:
I think we should make use more of the comments functionality to guide the author of an issue, i.e. use `<!--` and `-->` tags. See also the web repository's issue template: https://raw.githubusercontent.com/wireapp/wire-desktop/dev/.github/issue_template.md

So instead of "TODO" and "yes/no" as text, I would keep it as html comments, so a person not deleting that text but just adding information doesn't lead to us constantly seeing TODO in the issue.